src/target/source/source_module.cc

```diff
@@ -119,7 +122,6 @@ class CSourceModuleNode : public runtime::ModuleNode {
       return PackedFunc(
           [sptr_to_self, this](TVMArgs args, TVMRetValue* rv) { *rv = this->const_vars_; });
     } else {
-      LOG(FATAL) << "Unknown packed function: " << name;
```

Comment:
Is this related?

Reply:
Not really but just a quick fix. I could remove this line and send it along with the JSON runtime if you think that's better.

Reply:
I think that would be neater. Even a 1 line PR would be fine.

Reply:
This would be needed. Your runtime could get unnecessarily stuck here for other queries.

Reply:
If it's a necessary but unrelated change, I think a separate PR is most appropriate.

Reply:
done